src/styles/choices.scss

```diff
@@ -152,6 +152,7 @@ $choices-icon-cross-inverse: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjEiI
   font-size: $choices-font-size-md;
   min-height: 44px;
   overflow: hidden;
+  box-sizing: border-box;
```

Comment:
Instead of applying this to specific selectors, could we not just apply to everything within the .choices class?

Something like:

```css
.choices * {
  box-sizing: border-box;``
}
```

Reply:
Yes, we can apply to everything selectors, but we will lose a little performance.
I noticed that this is necessary for two classes.

Reply:
Universal selectors are slow and box-sizing is inheriting to children, so, agree with @dios-oleg that those styles better be applied when needed.